### ISSUE TITLE
ReferenceError: Property 'React' doesn't exist

### ISSUE THREAD

**Issue by thespacemanatee** [marklawlor/tailwindcss-react-native#59]
Hello,

I followed the very simple instructions from the documentation but can't seem to get past this error.

<img src="https://user-images.githubusercontent.com/6837599/170292964-68f7e274-78b4-4751-98c3-a55152fea477.png" width=400 />

### Repro

```ts
const App: React.FC = () => {
  return (
    <TailwindProvider> // <-- can't get past this
      <SafeAreaProvider>
        <Provider store={store}>
          <PersistGate loading={null} persistor={persistor}>
            <Root />
          </PersistGate>
        </Provider>
      </SafeAreaProvider>
    </TailwindProvider>
  );
};
```

Any ideas? Would really love to be able to use this library, thank you!

Running on:
```
expo@45.0.4
tailwindcss-react-native@1.4.0/1.5.0@next  // <--tried both
```

**Comment by jmeistrich** [comment]
I was getting that too when I was on an old version of Expo. Upgrading Expo solved it for me. I think it's because React 17 added the feature that importing React isn't necessary so React 16 would have this error. The library should import React to avoid this on old versions of RN?

**Comment by jmeistrich** [comment]
Oh interesting, I don't know then. I'll leave it to @marklawlor to answer :). But for what it's worth I'm using it with React 17 with no problem.

**Comment by thespacemanatee** [comment]
@jmeistrich 
Added this diff:

```diff
diff --git a/node_modules/tailwindcss-react-native/dist/provider.js b/node_modules/tailwindcss-react-native/dist/provider.js
index 0eec7af..16c9c5b 100644
--- a/node_modules/tailwindcss-react-native/dist/provider.js
+++ b/node_modules/tailwindcss-react-native/dist/provider.js
@@ -1,6 +1,8 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.TailwindProvider = void 0;
+import React from "react"
+
 const react_1 = require("react");
 const react_native_1 = require("react-native");
 const hooks_1 = require("@react-native-community/hooks");
```

and it works now, although I'm sure this is not intended. Will wait for the maintainer to revert but in the mean time we will use this solution first!

**Comment by thespacemanatee** [comment]
Enabling JSX transform seems to have resolved the issue for me, but this is kind of a huge gotcha that should be addressed in the documentation, or simply adding `import React from "react"` would be a good solution IMO since not everyone is using that babel setting. Feel free to close this issue!

**Comment by marklawlor** [comment]
Fixed in `1.4.2`. 

I'm going to keep the docs as-is. Out of the 3 common ways to create a RN application (`create-react-native-app`, `expo-cli`, `react-native init`) only 1 still uses the classic transform. In addition, web frameworks now have linter warnings when using the classic transform. As it's more likely that new users will be using the automatic transform, I'll keep the docs written that way.

At least now the code won't crash :)
